drivers/soundwire/intel.h

```diff
@@ -5,17 +5,20 @@
 #define __SDW_INTEL_LOCAL_H
 
 /**
- * struct sdw_intel_link_res - Soundwire link resources
```

Comment:
@plbossart again commenting on the commit message "This patches handles...." -> "This patch handles"

Reply:
done and simplified